digital-deck-server/services/cardDistService.js

```diff
@@ -0,0 +1,58 @@
+var Player = require("../services/player.js");
+var Table = require("../services/table.js");
+
+module.exports = {
+  /**
+   * Distributes the cards to the players based on the params.
+   *
+   * @param decks The number of decks to be used in the game session.
+   * @param players The number of players in a session.
+   * @param cardsPerPlayer The number of cards to give per player.
+   * @param cardsOnTable The number of cards to put on the table.
+   * @return {json} The json object containing players with player ids and cards assigned to them, table and cards of table and remaining cards on the deck.
+   */
+  distCards: function (decks, players, cardsPerPlayer, cardsOnTable) {
+    playerObjs = [];
+    table = new Table([]);
+
+    cards = getCards(decks);
```

Comment:
Shouldn't this be in `session.js` since the players would need to access the same deck of card throughout the session?

Reply:
Nvm, you're returning the value as `deck` so that's fine.